packages/ui/src/components/Layout/LayoutPage.tsx

```diff
@@ -10,8 +10,38 @@ import { ThemeScheme } from './Types'
 
 export const PageLayoutContent = ({ children }: { children: ReactNode }) => {
 	const prefix = useClassNamePrefix()
+	const [isOverflowing, setIsOverflowing] = useState(false)
+	const contentRef = useRef<HTMLDivElement>(null)
 
-	return <div className={`${prefix}layout-page-content`}>
+	useEffect(() => {
+		const contentRefCopy = contentRef.current
```

Comment:
can't you just add an early return if contentRefCopy is empty?

Reply:
you can now remove all optional chaining operator `contentRefCopy?.`

Reply:
Done.